generated/proto/metricpb/composite.proto

```diff
@@ -48,6 +48,11 @@ message ForwardedMetricWithMetadata {
   ForwardMetadata metadata = 2 [(gogoproto.nullable) = false];
 }
 
+message TimedMetricWithMetadata {
+  TimedMetric metric = 1 [(gogoproto.nullable) = false];
```

Comment:
What's the difference between ForwardedMetric and TimedMetric?  They seem the same except one has multiple values instead of a single value.